cc2538-bsl.py

```diff
@@ -784,11 +784,12 @@ def read(filename):
                 length = length + (length % 4)
```

Comment:
Looks like I made a mistake here in the past. This should be `length = (length + 4) - (length % 4)` I think. 
It doesn't really matter because of the `length >> 2` further down, but this makes the debug msg correct. And it makes sure we always round up when the user gives us funny input (aka not a multiple of 4).